docker/openemr/flex-3.7/autoconfig.sh

```diff
@@ -183,12 +186,14 @@ if [ -f /etc/docker-leader ] ||
         cd /var/www/localhost/htdocs
     fi
 
-    if [ -f /var/www/localhost/htdocs/auto_configure.php ]; then
+    if [ -f /var/www/localhost/htdocs/auto_configure.php ] &&
+       [ "$EASY_DEV_MODE" != "yes" ]; then
         chmod 666 /var/www/localhost/htdocs/openemr/sites/default/sqlconf.php
         chmod 666 /var/www/localhost/htdocs/openemr/interface/modules/zend_modules/config/application.config.php
-        chown -R apache /var/www/localhost/htdocs/openemr/
     fi
 
+    chown -R apache /var/www/localhost/htdocs/openemr/
```

Comment:
hi @d3sandoval ,
To be safe, this line should only run if /var/www/localhost/htdocs/auto_configure.php exists (to avoid running it again and again on docker compose stop/start). I am happy to make this change, just want to make sure you don't see it breaking anything for your stuff.

Reply:
(just let me know; if it's ok, then I'll merge in this PR and then make my change on a commit following)